layouts/download/list.html

```diff
@@ -1,99 +1,128 @@
 {{ define "main"}}
+<div id="course-popup" class=" popup">
+    <div class="popup-content">
+        <button id="btn-popup-close" class=" btn-popup-close">&times;</button>
+        <img class="popup-img" src="/images/orange_title_scaled.png" alt="">
+        <h2>NEW: Data Science email course [FREE]</h2>
+        <p>45% of Orange users want to learn more about data science (survey, 2017, n=6000). We want to help. Join
+            out email course and get weekly tips.</p>
```

Comment:
out email course - our email course

Reply:
I'm nitpicking, but I'd change the last sentence to 'Join our email course for weekly Orange tips and tricks!'